git/foundations_git/commit_messages.md

```diff
@@ -20,14 +20,47 @@ Yes! Let me give you a quick list of reasons why:
 
 -   Having a good commit message history will also be helpful if you come back to a project you were working on after stepping away from it for a while. You likely won't remember your thought process and changes made when initially writing out the code. 
 
+### Bad vs. Good Commits
+
+When it comes to writing commits, it is crucial to know how to write them effectively. Here's an example of a bad commit message:
+
+`fix a bug`
```

Comment:
Nit: Can we change this from inline code formatting to a full code block please? it's a bit more readable and stands out better.

```suggestion
~~~
fix a bug
~~~
```

It'll look like this on the site:
<img width="839" alt="Screenshot 2023-06-28 at 00 54 25" src="https://github.com/TheOdinProject/curriculum/assets/7963776/885d0021-c763-4d89-9a71-516e53fb0c8b">

As opposed to the inline code formatting:
<img width="631" alt="Screenshot 2023-06-28 at 00 53 51" src="https://github.com/TheOdinProject/curriculum/assets/7963776/773ef16d-34ca-4306-bef5-e4d8c890ccd2">